src/plugin/routes/PluginRoutes.tsx

```diff
@@ -8,18 +8,42 @@
 import React, { lazy, Suspense } from 'react';
 import { Route } from 'react-router-dom';
 import { usePluginRoutes } from '../hooks';
+import { dynamicImportPlugin } from '../utils';
 import type { IRouteExtension } from '../types';
 
-interface PluginRoutesProps {
+interface IPluginRoutesProps {
   userPermissions?: string[];
   isAdmin?: boolean;
   fallback?: React.ReactElement;
 }
 
+const ERROR_FALLBACK_STYLES = {
+  container: {
+    padding: '40px',
+    textAlign: 'center' as const,
+    backgroundColor: '#f8f9fa',
+    border: '1px solid #dee2e6',
```

Comment:
These need to be configured in the CSS file. We don't hard code colors in the code base